www/PoyntPlugin.js

```diff
@@ -12,6 +12,50 @@ var Poynt = {
 				"amount": amount, 
 				"referenceId": referenceId
 			}]);
+	},
+	launchBilling: function (paymentid, successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchBilling', 
+			[{
+				"paymentid": paymentid
+			}]);
+	},
+	launchAskConf: function (msg,successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchAskConf', 
+			[{"msg": msg}]);
+	},
+	launchSign: function (title,butt,msg,successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchSign', 
+			[{"msg": msg,"title":title,"leftbutton":butt}]);
+	},
+	launchInit: function (successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchInit', 
+			[]);
+	},
+	launchMsg: function (msg,successCallback, errorCallback){
```

Comment:
Please:
1. rename to `displayMessage` to keep consistent with the Android interface
2. put this function inside `secondScreen` object
3. put `msg` inside `options`